build.gradle.kts

```diff
@@ -41,8 +41,7 @@ measureBuilds {
 spotless {
     kotlin {
         target("**/*.kt")
-        targetExclude("$buildDir/**/*.kt")
-        targetExclude("bin/**/*.kt")
+        targetExclude("$buildDir/**/*.kt", "bin/**/*.kt", "buildSrc/build/**/*.kt", "vendor/bundle/**/*.kt")
```

Comment:
When committing with GitKraken ktlint was picking up Kotlin files in build folders, this fixed it.

Reply:
It can be simplified this way.

https://github.com/MiSikora/laboratory/blob/eb7b62914cc65fa53590aaa51976f2351c505710/build.gradle.kts#L46-L51

Reply:
That's a good idea. I have just included the folders with `src` in them. I was going to do `**/src/**/*.kt`, but the build folder has a `src` folder.

Reply:
Ah, sorry. I gave you a wrong suggestion. In my projects I typically configure spotless per project and not globally. Something like this: https://github.com/MiSikora/laboratory/blob/ec64a5221a48bf0caa93f77b3c042117b9d0ce8c/build.gradle.kts#L42-L77

But now I noticed that in Pocket Casts we configure it globally. So exclusions that you configured before were correct. Because currently Spotless doesn't work for `modules` as there are no directories like `modules/src`.